nova.mix.js

```diff
@@ -0,0 +1,33 @@
+const mix = require('laravel-mix')
+const webpack = require('webpack')
+const path = require('path')
+
+class NovaExtension {
+  name() {
+    return 'nova-extension'
+  }
+
+  register(name) {
+    this.name = name
+  }
+
+  webpackConfig(webpackConfig) {
+    webpackConfig.externals = {
+      vue: 'Vue',
+    }
+
+    webpackConfig.resolve.alias = {
+      ...(webpackConfig.resolve.alias || {}),
+      'laravel-nova': path.join(
+        __dirname,
+        '../../vendor/laravel/nova/resources/js/mixins/packages.js'
```

Comment:
as we are outside of a `nova-components` directory the correct path is `'vendor/laravel/nova/resources/js/mixins/packages.js'` explained [here](https://nova.laravel.com/docs/customization/frontend.html#using-nova-mixins)

Reply:
Fixed, thank you.